transactions/create_treasury.cdc

```diff
@@ -3,7 +3,7 @@ import FlowToken from "../contracts/core/FlowToken.cdc"
 import FiatToken from "../contracts/core/FiatToken.cdc"
 import FUSD from "../contracts/core/FUSD.cdc"
 
-transaction(initialSigners: [Address], initialThreshold: UInt64) {
+transaction(initialSigners: [Address], initialThreshold: Int) {
```

Comment:
It's possible to create treasury with threshold 0

Reply:
Also posible to add negative values - when we use int instead of uint

Reply:
ahh true, i'll just use a regular `UInt` then. instead of 64 bit uint